lib/resty/mlcache.lua

```diff
@@ -269,6 +275,20 @@ local function set_lru(self, key, value, ttl, neg_ttl)
         ttl = nil
     end
 
+    if l1_serializer then
+        local ok, err
+        ok, value, err = pcall(l1_serializer, value)
+        if not ok then
+            return nil, "l1_serializer threw an error: " .. value
+        elseif value == nil then
+            if err then
+                return nil, "l1_serializer returned an error: " .. err
+            else
+                return nil, "l1_serializer returned a nil value"
```

Comment:
Done. Actually it is not strictly necessary and wasn't the case initially: it has been added with 0e917465c1e8cc8b6c406ecdc87fa9092e8489da (because is made my troubleshooting more complex at some point, and makes no sense in general I think).